test/bin/scenario.sh

```diff
@@ -196,7 +199,15 @@ prepare_kickstart() {
               -e "s|REPLACE_REDHAT_AUTHORIZED_KEYS|${REDHAT_AUTHORIZED_KEYS}|g" \
               -e "s|REPLACE_PUBLIC_IP|${PUBLIC_IP}|g" \
               -e "s|REPLACE_FIPS_COMMAND|${fips_command}|g" \
+              -e "s|REPLACE_ENABLE_MIRROR|${ENABLE_MIRROR}|g" \
               > "${output_file}"
+    if [ "${ENABLE_MIRROR}" ]; then
+        local -r mirror_cert_content=$(awk '{printf "%s\\n", $0}' "${cert}")
```

Comment:
The `$cert` file might not exist. We need to check this.
Also, can we log OK / SKIPPED on mirroring using junit?

Reply:
Added a check for the cert.